forms/static/forms/admin/dependent_fields.js

```diff
@@ -50,62 +50,70 @@
-            long_monitoring_mode()
+            long_monitoring_mode();
         }
     });
     $(document).change(function() {
-        const removeValue = function(id) {
-            $(`#${id}`).children('option:first').removeAttr('value');
-        };
-        const addValue = function(id) {
+        const removeValidation = function(id) {
             $(`#${id}`).children('option:first').attr('value', '');
         };
+        const addValidation = function(id) {
+            // Set --- since it's an invalid value and thus will receive validation error if user doesn't select another value
+            $(`#${id}`).children('option:first').attr('value', '----')
+        };
         let newspaper_person = $('#id_newspaperperson_set-0-victim_or_survivor_1:checked').length;
         let radio_person = $('#id_radioperson_set-0-victim_or_survivor_1:checked').length;
         let tv_person = $('#id_televisionperson_set-0-victim_or_survivor_1:checked').length;
         let internent_person = $('#id_internetnewsperson_set-0-victim_or_survivor_1:checked').length;
 
-        if((newspaper_person > 0) || (radio_person > 0) || (tv_person > 0) || (internent_person > 0)){
-            addValue('id_newspaperperson_set-0-victim_of');
-            addValue('id_newspaperperson_set-0-survivor_of');
-            addValue('id_radioperson_set-0-victim_of');
-            addValue('id_radioperson_set-0-survivor_of');
-            addValue('id_televisionperson_set-0-victim_of');
-            addValue('id_televisionperson_set-0-survivor_of');
-            addValue('id_internetnewsperson_set-0-victim_of');
-            addValue('id_internetnewsperson_set-0-survivor_of');
-            $('.field-victim_of').hide();
-            $('.field-survivor_of').hide();
-            $('.field-is_photograph').hide();
-            $('.field-special_qn_1').hide();
-            $('.field-special_qn_2').hide();
-            $('.field-special_qn_3').hide();
-            $('.field-item_number').hide();
-            $('.field-webpage_layer_no').hide();
-        }else{
-            removeValue('id_newspaperperson_set-0-victim_of');
-            removeValue('id_newspaperperson_set-0-survivor_of');
-            removeValue('id_radioperson_set-0-victim_of');
-            removeValue('id_radioperson_set-0-survivor_of');
-            removeValue('id_televisionperson_set-0-victim_of');
-            removeValue('id_televisionperson_set-0-survivor_of');
-            removeValue('id_internetnewsperson_set-0-victim_of');
-            removeValue('id_internetnewsperson_set-0-survivor_of');
-            $('.field-victim_of').show();
-            $('.field-survivor_of').show();
         const monitor_mode = $("#id_monitor_mode").find(":selected").val()
         if(monitor_mode === "2"){
             short_monitoring_mode()
         }else{
             long_monitoring_mode()
         }
+
+        if((newspaper_person > 0) || (radio_person > 0) || (tv_person > 0) || (internent_person > 0)){
```

Comment:
@ascii-dev  kindly help manually test this to ensure everything is working as expected

Reply:
On it... @esirK

Reply:
@esirK it works as expected... ✅